Add image to Roborock (#29574)
home-assistant/home-assistant.github.io: source/_integrations/roborock.markdown

Volume - This allows you to control the volume of the robot's voice. For example, when it states "Starting cleaning". This allows you to set the volume to 0%, while the app limits it to 20%.

### Image

> You are able to see all of of the maps within your roborock account - keep in mind they are device specific. The maps require the cloud api in order to communicate as the maps are seemingly stored on the cloud (if someone can figure out a way around this - contributions are always welcome.)

Reviewer: ```suggestion
You can see all the maps within your Roborock account. Keep in mind that they are device-specific. The maps require the cloud API to communicate as the maps are seemingly stored on the cloud. If someone can figure out a way around this - contributions are always welcome.
```
